docs/marketplace/github-marketplace/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: GitHub Marketplace
+description: Appcircle GitHub action allows you to distribute your builds to testers directly pipeline.
+tags:
+  [
+    github-marketplace,
+    testing-distribution,
+    ipa distribution,
+    apk distribution,
+    binary distribution,
+  ]
+sidebar_position: 2
+---
+
+Enhance power of distributing the builds to test groups or release beta version of your apps with using **Appcircle Testing Distribution** and **Appcircle Enterprise Store** actions
```

Comment:
Enhance the power of distributing builds to test groups or releasing beta versions of your apps by using the **Appcircle Testing Distribution** and **Appcircle Enterprise Store** actions.